src/main/java/com/digitaldan/jomnilinkII/OmniInvalidResponseException.java

```diff
@@ -9,13 +9,18 @@
  */
 package com.digitaldan.jomnilinkII;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class OmniInvalidResponseException extends Exception {
+	private final Logger logger = LoggerFactory.getLogger(OmniInvalidResponseException.class);
 
 	private Message response;
 
 	public OmniInvalidResponseException(Message response) {
 		super();
 		this.response = response;
+		logger.error("Invalid message type received: {}", response.toString());
```

Comment:
Since this is an exception class, can you remove the logging from it? Who ever gets this should do the logging.  Also i avoid logging anything other then trace/debug/warn since this is a shared library and I don't want to pollute the calling application with logging under normal conditions.

Reply:
Yeah, that was supposed to be removed. I had added it to help myself figure out what was going on but forgot to remove it.